### PR TITLE
feature: build committer server

### LINKED ISSUE #55: committer server
Create a committer server to inscribe data on block about 10 mins once

### PR DESCRIPTION
**Pull request by zhouop0**
Closes: #55 

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: bitcoin/committer_test.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+	"github.com/btcsuite/btcd/rpcclient"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewCommitterRpcClient(t *testing.T) {
+	connCfg := &rpcclient.ConnConfig{
+		Host:         "43.157.135.191:8332/wallet/test",
+		User:         "b2node",
```
Comment (0x677261706562616261) [inline bitcoin/committer_test.go]: use env variable instead of plain text

File: bitcoin/committer_test.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		Pass:         "b2node@pass",
+		HTTPPostMode: true,
+		DisableTLS:   true,
+	}
+	client, err := rpcclient.New(connCfg, nil)
+	require.NoError(t, err)
+	require.NotEmpty(t, client)
+}
+
+//func TestNewInscriptionTool(t *testing.T) {
```
Comment (0x677261706562616261) [inline bitcoin/committer_test.go]: what is the problem with it?

File: bitcoin/commiter_service.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+	"crypto/rand" // #nosec G702
+	"time"
+
+	"github.com/tendermint/tendermint/libs/service"
+)
+
+const (
+	BitcoinServiceName = "BitcoinCommiterService"
+
+	WaitTimeout = 1 * time.Minute
```
Comment (0x677261706562616261) [inline bitcoin/commiter_service.go]: 1 or 10 min?

File: bitcoin/import_descriptors_cmds.go [hunk 4]
```diff
@@ -0,0 +1,76 @@
+package bitcoin
+
+import (
+	"encoding/json"
+
+	"github.com/btcsuite/btcd/btcjson"
+	"github.com/btcsuite/btcd/rpcclient"
+)
+
+type Descriptor struct {
```
Comment (0x677261706562616261) [inline bitcoin/import_descriptors_cmds.go:11]: what does this struct represent?

**Comment by zhouop0** [comment]
go-version: 1.20.5
actions/setup-go@v4.1.0

**Pull request merged**